### ISSUE TITLE
Multi-currency unit may be incorrect

### ISSUE THREAD

**Issue by arjanpot** [Meaningful-Data/xbridge#5]
Hi, I expect the "unit" column in the csv output to contain an ISO code for the currency.

For example in report c_72.00.w.csv of COREP LCR DA,

_expected is:_
eba_CU:AED,dp146854,6739000,_**iso4217:AED**_

_instead of (xbridge output):_
eba_CU:AED,dp146854,6739000,_**uAED**_.

Converted sample instance file like ---CON_FR_COREP030100_COREPLCRDA_2023-12-31---.XBRL provided by EBA.

Please let me know if my assumption is correct and if this can be fixed in xbridge.
Thanks!

**Comment by antonio-olleros** [comment]
Confirmed, it is as @arjanpot says, so we'll proceed to fix it

**Comment by antonio-olleros** [comment]
Hi @arjanpot , I just did a pull request to fix this issue. We are assuming that always

- The unitRef in the XML file will start with a u (like uEUR). We have seen it like that in the sample files. 
- When there is a unit attribute in a CSV file, it will always be a currency

If these assumptions are correct, then we just need to replace the u with the iso4217 prefix

**Comment by arjanpot** [comment]
In practice these assumptions will work for the EBA samples. It may not work when the xbrl file is produced by another tool.

**Comment by antonio-olleros** [comment]
Can you please specify why? I mean, is it because the u is used in the EBA samples, but other tools may use something different?

**Comment by arjanpot** [comment]
As an example:

```
unitRef="uEUR"
```

is referencing

```
  <xbrli:unit id="uEUR">
    <xbrli:measure>iso4217:EUR</xbrli:measure>
  </xbrli:unit>

```

this could als be


```
unitRef="SOME_ID"
```

is referencing

```
  <xbrli:unit id="SOME_ID">
    <xbrli:measure>iso4217:EUR</xbrli:measure>
  </xbrli:unit>

```

**Comment by antonio-olleros** [comment]
You are completely right, sorry I missed that... I will correct it

**Comment by javihern98** [comment]
Fixed on #7
